tests/rptest/tests/node_folder_deletion_test.py

```diff
@@ -78,14 +78,17 @@ def test_deleting_node_folder(self):
         wait_until(lambda: producer.produce_status.acked > 100000,
                    timeout_sec=120,
                    backoff_sec=0.5)
-        to_stop = random.choice(self.redpanda.nodes)
+        # explicitly skip node 0 as this is a seed server and its id doesn't change
+        to_stop = random.choice(self.redpanda.nodes[1:])
```

Comment:
Another option would be to `set_seed_servers(redpanda.nodes)` before running this test